ingest/rules/curate.smk

```diff
@@ -78,6 +97,7 @@ rule curate:
                 --abbr-authors-field {params.abbr_authors_field:q} \
             | augur curate apply-geolocation-rules \
                 --geolocation-rules {input.geolocation_rules:q} \
+            | python {input.urls_script:q} \
             | augur curate apply-record-annotations \
                 --annotations {input.annotations:q} \
```

Comment:
This no longer works – the accessions in [ingest/defaults/annotations.tsv](https://github.com/nextstrain/mpox/blob/f9c1b689e145a6ffb4ffa3111b2d0f028e466582/ingest/defaults/annotations.tsv) haven't been updated to Pathoplexus counterparts.